knowledge-base/chart-datatable-binding.md

```diff
@@ -0,0 +1,78 @@
+---
+title: How to use DataTable as source of data for the Chart
+description: How to convert DataTable to List and use it as data source for the Chart
+type: how-to
+page_title: How to use DataTable as data source for the Chart
+slug: chart-datatable-binding
+position: 
+tags: 
+ticketid: 1589460 
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Charts for Blazor</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+I want to use DataTable as a data source for the Chart series.
```

Comment:
It is acceptable to say the same thing 1-2 times, so that you include more / all desired keywords.
```suggestion
How to data bind the Telerik Blazor Chart to a DataTable?

How to use a DataTable as a data source for the Chart?
```